src/utils.rs

```diff
@@ -523,6 +523,8 @@ pub fn gradient_check<F>(fn_closure: F, input: &Array, eps: f64, grad: &Array) -
   let num_grad = numerical_gradient(fn_closure, input, eps);
   println!("numgrad = {:?}\ngrad = {:?}", array_to_vec(&num_grad), array_to_vec(grad));
 
+  af::print(&num_grad);
+  af::print(&grad);
```

Comment:
Do not commit debug print statements! You can also use `AF_DISABLE_GRAPHICS=1 RUST_TEST_THREADS=1 cargo test -- --nocapture` to display the results of the gradient tests as listed in the README.